pep.rst

```diff
@@ -240,6 +240,21 @@ Therefore these prefixes can't be used as a tag:
 
     bytesprefix: "b" | "B" | "br" | "Br" | "bR" | "BR" | "rb" | "rB" | "Rb" | "RB"
 
+A tag name can also be any *dotted* name, for example ``app.html``. By extension, this also
```

Comment:
There's also the sentence `The tag name can be any *undotted* name that isn't already an existing valid
string or bytes prefix` 12 lines above this one that should be changed.

Reply:
Given dotted names, I do like the idea of `lazy.f` - it seems much better than the previous alternative we were considering, `fl`.

Reply:
`lazy.f` sounds like a good name indeed, it's bound to confuse people though. Not sure.

Reply:
> The first mention has can be. The second then says can also be. I hoped to kind of pair them: simple case, advanced case. If that's confusing, I can change.

Maybe say `any name` in the first one? Or `any undotted name`, but remove the emphasis?

Reply:
I changed to say `any name`. Then in the second mention: "This means a tag name can be a dotted name, for example ``app.html``."